### DISCUSSION TITLE
Integrating menu-driven bot with portal or custom bot client

### DISCUSSION THREAD

**Discussion by aayushkedia95** [Samagra-Development/SamagraX-community#6] [Troubleshooting and Clarifications]
How can we integrate menu-driven workflow with portal bot client or custom bot client? In sunbird we have a separate bot client for the portal, so how can we integrate with it?

@ChakshuGautam @nik720

**Comment by aayushkedia95** [comment]
@Nik720, we will write a Wiki on this and share.

**Reply by Nik720** [reply]
@aayushkedia95 Is wiki documentation is ready?

cc - @vinukumar-vs @vrayulu

**Reply by aayushkedia95** [reply]
@Nik720, we will be updating this by Friday EoD. Hope that helps. 

Let us know if you would need it earlier.

**Reply by aayushkedia95** [reply]
@Nik720, this will happen through creation of a new adapter. Just wanted to inform you that we are creating a new adapter for telegram towards the end of April and will be documenting the process alongside. We will share the same after that.
